deployer/manager.go

```diff
@@ -39,7 +39,13 @@ func NewDeploymentManager(sub substratemanager.Manager, nodeClient *client.NodeC
 	}
 }
 func (d *deploymentManager) CancelAll() {
```

Comment:
i think you should first cancel the deployments' contracts, then delete them locally.